cvat-core/src/common.ts

```diff
@@ -145,3 +146,17 @@ export function filterFieldsToSnakeCase(filter: Record<string, string>, keysToSn
 export function isResourceURL(url: string): boolean {
     return /\/([0-9]+)$/.test(url);
 }
+
+export function isPageSize(value: number | 'all'): boolean {
+    return isInteger(value) || value === 'all';
+}
+
+export function fieldsToSnakeCase(
+    params: Record<string, any>, out?: Record<string, any>,
+): Record<string, any> {
```

Comment:
I do not think this method needs the second parameter
It should accept one object and return another one, modifying function parameters is not pure approach.